build-tools/Xamarin.Android.Tools.BootstrapTasks/result-packaging.targets

```diff
@@ -8,6 +8,7 @@
     <_BuildStatusFiles Include="$(BootstrapOutputDirectory)XABuildConfig.cs" Condition="Exists('$(BootstrapOutputDirectory)XABuildConfig.cs')" />
     <_BuildStatusFiles Include="$(BootstrapOutputDirectory)*.binlog" />
     <_BuildStatusFiles Include="$(BootstrapOutputDirectory)prepare*.log" />
+    <_BuildStatusFiles Include="$(BootstrapOutputDirectory)*.json" />
```

Comment:
This line appears wrong, in that the generated `xa-build-status-11.1.99.16_8105993-Darwin-Release.zip` doesn't contain any `CGManifest.json` files. :-(

Reply:
Lol, I never added `Step_GenerateCGManifest` to any scenario! Thus, it never ran!

Reply:
Now it's in there.